src/service/resources-impl.js

```diff
@@ -342,9 +342,6 @@ export class Resources {
 
     // Create and add the resource.
     const resource = new Resource((++this.resourceIdCounter_), element, this);
-    if (!element.id) {
-      element.id = 'AMP_' + resource.getId();
```

Comment:
This might be useful to keep in dev mode. I remember chatting about this with @dvoytenko